### DISCUSSION TITLE
migrate-disqus-github-discussions-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [estruyf/eliostruyf-comments#608] [Comments]
# migrate-disqus-github-discussions-giscus/

Azure / Microsoft 365 / Visual Studio Code / Speaker / Blogger

https://www.eliostruyf.com/migrate-disqus-github-discussions-giscus/

**Comment by mirontoli** [comment]
I heard about `giscuss` when I listened to your talk last Wednesday at ESPC22. I visited your blog, and voilà, here is more information. Thank you.

**Reply by estruyf** [reply]
Cool isn't it? So happy that I finally moved to it.

**Comment by tejasghlade** [comment]
its cool
